packages/block-editor/src/components/url-input/index.js

```diff
@@ -179,6 +183,11 @@ class URLInput extends Component {
 					}
 					break;
 				}
+				case ENTER: {
+					event.preventDefault();
```

Comment:
What is this preventing

Reply:
Not sure, for the current enter keypress code we were calling  "event.preventDefault();" so I followed similar logic. But, it seems the calls to event.preventDefault(); are not needed so I removed them.